client/containers/PipelineContainer.tsx

```diff
@@ -13,7 +13,8 @@ export default class PipelineContainer extends ChildContainer<
     super();
     this.state = {
       currentPartnerFirstName: 'ALL',
-      currentTeam: 'default',
+      // current user's team
+      currentTeam: 'All',
```

Comment:
Need a way to access the current user -> their partnerTeam

My q:
In pipeline container, I want to initialize the currentTeam state on the container to the current user's partnerTeam. Can I access the current user container directly from pipeline container  to get user.partnerTeam in pipeline container (& how?)? Or, do I need to update this from componentDidMount() in Kanban?

Do we have access to all other containers in any given container? How to access?

Ideas:
- [Vinai] When the user logs in and the user containers updates it stated, It might make sense to concurrently update the pipeline state
-

Reply:
@KNWR Kanban.tsx you have access to the user object via the props. Then you can update the state of the Pipeline Container as well as load the appropriate companies in componentDidMount()